src/Server.php

```diff
@@ -21,8 +21,6 @@ interface Server
      *
      * @param  string $function
      * @param  string $namespace
-     * @param  null|array Optional array of arguments to pass to callback at
```

Comment:
I'm not sure this should be dropped. This is a pseudo optional parameter that was introduced in a BC compatible way.
On next major new optional parameter should be added to the method.

Reply:
Ah, you are right! I thought it was mistake as there was no variable name, but you are totally correct. We probably need to add `array $argv = []` in next major.